pytests/test_wff_min_fam_tax_credit.py

```diff
@@ -0,0 +1,56 @@
+from . import Reasoner
+
+
+class TestKey(Reasoner):
+    key = 'isWorkingForFamiliesMinimumFamilyTaxCredit'
+
+
+class TestWFFMinFamTaxCreditDefault(TestKey):
+
+    """
+    Benefit: Working for Families - Minimum Family Tax Credit (default state):
+    Default benefit.isWorkingForFamiliesMinimumFamilyTaxCredit is FORBIDDEN
+    Overriden by: Benefit: Working for Families
+     - Minimum Family Tax Credit (eligibility)
+    """
+
+    body = {}
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+        self.assertTrue(self.is_conclusive)
+
+
+class TestWFFMinFamTaxCreditElig(TestKey):
```

Comment:
@Br3nda Can you please take a look at this test that is failing, should be working, but I'm also getting Forbidden in Raap.